pkg/server/chaosd/disk.go

```diff
@@ -150,11 +152,34 @@ func (s *Server) DiskFill(fill *core.DiskCommand) (uid string, err error) {
 	}
 
 	var cmd *exec.Cmd
+	if fill.Size != "" {
+		fill.Size = strings.Trim(fill.Size, " ")
+	} else if fill.Percent != "" {
+		fill.Percent = strings.Trim(fill.Percent, " ")
+		percent, err := strconv.ParseUint(fill.Percent, 10, 0)
+		if err != nil {
+			log.Error(fmt.Sprintf(" unexcepted err when parsing disk percent '%s'", fill.Percent), zap.Error(err))
+			return uid, err
+		}
+		dir, err := os.Getwd()
```

Comment:
Do we need to consider `file.Path` here?

Reply:
do you mean `fill.Path`?

Reply:
Yes, typo has been fixed 😂

Reply:
Ok, I agree with you.